### PR TITLE
feat: add godwoken mainnet addresses

### PR DESCRIPTION
**Pull request by Kuzirashi**
Based on: https://github.com/safe-global/safe-deployments/pull/147

### CODE REVIEW

File: gnosis/eth/clients/blockscout_client.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ class BlockscoutClient:
         EthereumNetwork.REI_CHAIN_TESTNET: "https://scan-test.rei.network/",
         EthereumNetwork.METER_MAINNET: "https://scan.meter.io/",
         EthereumNetwork.METER_TESTNET: "https://scan-warringstakes.meter.io/",
-        EthereumNetwork.GODWOKEN_TESTNET_V1: "https://v1.betanet.gwscan.com/",
+        EthereumNetwork.GODWOKEN_TESTNET_V1: "https://v1.testnet.gwscan.com/",
```
Comment (moisses89) [inline gnosis/eth/clients/blockscout_client.py:43]: Is this blockscout api compatible?

**Review by moisses89 — COMMENTED** [review]
Could you verify the contracts in the blockexplorer?

**Comment by Uxio0** [comment]
@Kuzirashi are you still interested in merging this?

File: gnosis/eth/clients/blockscout_client.py (hunk 1 above)
Comment (Kuzirashi) [reply gnosis/eth/clients/blockscout_client.py:43]: I'm not sure and I don't have time to test it unfortunately. I just updated old one which is probably deprecated to new one.

**Comment by Kuzirashi** [comment]
> @Kuzirashi are you still interested in merging this?

I think it would be an improvement to the current codebase because new network would actually work. Now it doesn't.

File: gnosis/eth/clients/blockscout_client.py (hunk 1 above)
Comment (Uxio0) [reply gnosis/eth/clients/blockscout_client.py:43]: Please make sure, if not remove it

**Comment by Uxio0** [comment]
> I think it would be an improvement to the current codebase because new network would actually work. Now it doesn't.

Sure, then try to be responsive here so we can fix the issues and merge it asap

**Comment by Uxio0** [comment]
Close due to inactivity

**Pull request closed without merging**